src/main/java/org/zalando/planb/provider/CassandraConfig.java

```diff
@@ -19,6 +20,7 @@ Session initializeCassandra() {
         final Cluster.Builder builder = Cluster.builder();
 
         builder.addContactPoints(cassandra.getContactPoints().split(","));
+        builder.withAddressTranslator(new EC2MultiRegionAddressTranslator());
```

Comment:
Had a quick look and it seems that apart from the `IdentityTranslator` (which does nothing) this is the only other available `AddressTranslator`.
Maybe it would be enough to have something more functional like a flag that defines the app as running on ec2. This actually tells me there should be an easy to check that automatically